index.js

```diff
@@ -0,0 +1,29 @@
+const { parse } = require('csv-parse')
+const fs = require('fs')
+const habitablePlanets = []
+
+function isHabitable (planet) { // checks if all the conditions to confirm that a planet is habitable are met

```

Comment:
let remove the comment here, we only need to write comments if the particular part of the code is complex and needs explanation. The function is named well so the comment isn't needed